src/lib.rs

```diff
@@ -769,6 +764,10 @@ mod tests {
         assert_eq!(Mime::from_str("text/plain; charset=utf-8").unwrap(), TEXT_PLAIN_UTF_8);
         assert_eq!(Mime::from_str("text/plain;charset=\"utf-8\"").unwrap(), TEXT_PLAIN_UTF_8);
 
+        // quotes + semi colon
```

Comment:
This ‘addition’ seems to be an artifact of resolving a merge conflict on my side. LGTM.